src/components/Network.js

```diff
@@ -33,27 +34,45 @@ class Network extends React.Component {
       connected_peers: [],
       known_peers: [],
       loaded: false,
-      isLoading: false
+      isLoading: false,
+      peers: [],
+      peer_id: '',
     }
 
     this.loadData = this.loadData.bind(this);
   }
 
   componentDidMount() {
-    this.loadData();
-    this.loadTimer = setInterval(this.loadData, 1000);
+    const { match: { params } } = this.props;
+
+    networkApi.getPeers().then((peers) => {
+      this.setState({peers}, () => {
+        let peer_id = params.peer_id;
+        if (!peer_id) {
+          peer_id = peers[0];
+        }
+        this.props.history.push(`/network/${this.state.peers[0]}`);
+        this.setState({peer_id}, () => {
```

Comment:
Add comment explaining that you will load the data immediately and also schedule an update every second.